packages/cross-chain-core/src/CrossChainCore.ts

```diff
@@ -0,0 +1,60 @@
+import { Account, Network } from "@aptos-labs/ts-sdk";
+
+import {
+  WormholeInitiateTransferRequest,
+  WormholeInitiateTransferResponse,
+  WormholeProvider,
+  WormholeQuoteRequest,
+  WormholeQuoteResponse,
+} from "./providers/wormhole";
+
+export interface CrossChainDappConfig {
+  network: Network;
+  disableTelemetry?: boolean;
+}
+
+export type AptosAccount = Account;
+
+export type Chain = "Solana" | "Ethereum" | "Sui" | "Aptos";
+
+export type CCTPProviders = "Wormhole";
+
+export type UsdcBalance = {
+  amount: string;
+  decimal: number;
+  display: string;
```

Comment:
Not sure I would implement `display` in this component. I think better to let the developer choose how they want to display. I think it just gets complicated because

1. Developer might want to only use period to dilineate decimals, while others may want to use comma
2. Developer might want to use some kind of significant figures logic

Just think it's not a good idea to put it in this package because it is hard to undo these decisions later